[#1252] Add front-end tests reminder on Developer Guide (#1384)
reposense/RepoSense: docs/dg/workflow.md

Read [Cypress's Documentation](https://docs.cypress.io/api/commands/document.html#Syntax) to familiarize yourself with its syntax and [Cypress's debugging guide](https://docs.cypress.io/guides/guides/debugging.html#Log-Cypress-events) to tackle problems with your tests.
</box>

<box type="warning" seamless>

> Note that it is **compulsory** to add tests for the new front end changes that you made in order to prevent regression bugs, except for trivial changes that are unlikely to cause any regression, or for other situations where testing is not applicable for the change.

Reviewer: ```suggestion
Note that it is **compulsory** to add tests for the new front-end changes you made to prevent regression bugs, except for trivial changes that are unlikely to cause any regression or other situations where testing does not apply to the change..
```

Author: Thanks for the suggestion. I have made changes accordingly.
